gen3/bin/kube-setup-argo.sh

```diff
@@ -243,7 +243,7 @@ if [[ "$ctxNamespace" == "default" || "$ctxNamespace" == "null" ]]; then
 
     helm repo add argo https://argoproj.github.io/argo-helm --force-update 2> >(grep -v 'This is insecure' >&2)
     helm repo update 2> >(grep -v 'This is insecure' >&2)
-    helm upgrade --install argo argo/argo-workflows -n argo -f ${valuesFile}
+    helm upgrade --install argo argo/argo-workflows -n argo -f ${valuesFile} --version 0.22.11
```

Comment:
So instead of harcoding this version, let's try to read from manifest, with this version as a default. 

See karpenter script